libs/coin-evm/docs/logic.md

```diff
@@ -0,0 +1,44 @@
+# `logic.ts`
```

Comment:
There are already comments in the code itself, merge infos ?

Reply:
Same answer as above